arelle/plugin/xbrlDB/SqlDb.py

```diff
@@ -678,8 +678,8 @@ def getTable(self, table, idCol, newCols=None, matchCols=None, data=None, commit
   INSERT INTO %(table)s (%(newCols)s)
   SELECT %(newCols)s
   FROM row_values v''' + ('''
-  WHERE NOT EXISTS (SELECT 1 
-                    FROM %(table)s x 
+  WHERE NOT EXISTS (SELECT 1
```

Comment:
Calling this out as an example of trailing whitespace that does cause a difference at runtime. This example is inconsequential, but whoever QAs this needs to go through the diffs to check for others.

Reply:
Should we maybe break this down into smaller chunks, eg all files a-h and so on in separate PRs? Might make it a bit easier to review?

Reply:
This is okay. I prefer it as a single commit. I'll review/QA, I just need a chunk of time to go through it.

Reply:
OK, thanks!

Reply:
I reviewed and didn't find any problematic changes. I can merge once the merge conflicts are resolved, just don't rebase this one, that would make verifying again more difficult in this case.

Reply:
I resolved conflicts from within GitHub here on the web and just noticed the merge message was merge master into this branch. I think that’s specifically what you didn’t want. Really sorry about this mess up. 🤦‍♂️

Reply:
That's actually what I wanted. 🙂 The merge commit was easy to review. After the CI run passes I'll merge this.

Reply:
😮‍💨 thanks for reviewing this!